src/components/dialogs/AppDialogs.tsx

```diff
@@ -12,14 +12,12 @@ export default function AppDialogs() {
 
   return (
     <>
-      <EmailDialog
-        open={openDialog === "email"}
-        onOpenChange={onOpenChange("email")}
-      />
-      <EditProfileDialog
-        open={openDialog === "edit_profile"}
-        onOpenChange={onOpenChange("edit_profile")}
-      />
+      {openDialog === "email" && (
```

Comment:
changing these so that the component mounts only when the current session is already loaded

Reply:
smart